pkg/apis/core/validation/validation_test.go

```diff
@@ -14454,6 +14454,28 @@ func TestValidatePodStatusUpdate(t *testing.T) {
 		),
 		"",
 		"restartable init container can restart if RestartPolicyAlways",
+	}, {
```

Comment:
add a test case that we allow to set to the same value

Reply:
added.